### PR TITLE
[IOTDB-5502] Source counters for construct batches in IoTConsensus

### PR DESCRIPTION
**Pull request by BUAAserein**
#### Add two counters for construct batches in IoTConsensus. 
Please see details in [IOTDB-5502](https://issues.apache.org/jira/browse/IOTDB-5502).

### CODE REVIEW

**Review by neuyilan — APPROVED** [review]
LGTM

File: consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/Batch.java [hunk 1]
```diff
@@ -55,6 +57,9 @@ public void buildIndex() {
 
   public void addTLogEntry(TLogEntry entry) {
     logEntries.add(entry);
+    if (entry.fromWAL) {
+      logEntriesNumFromWAL += 1;
```
Comment (OneSizeFitsQuorum) [inline consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/Batch.java:61]: logEntriesNumFromWAL++

File: consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/LogDispatcher.java [hunk 2]
```diff
@@ -65,6 +66,9 @@ public class LogDispatcher {
 
   private boolean stopped = false;
 
+  private final AtomicLong logEntriesFromWAL = new AtomicLong(0);
+  private final AtomicLong logEntriesFromConsensusRequest = new AtomicLong(0);
```
Comment (OneSizeFitsQuorum) [inline consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/LogDispatcher.java:70]: FromConsensusRequest -> FromQueue

**Review by SpriCoder — CHANGES_REQUESTED** [review]
Excellent Work!
Just one problem, Please update the en and zh doc at same time
en: docs/UserGuide/Monitor-Alert/Metric-Tool.md
zh: docs/zh/UserGuide/Monitor-Alert/Metric-Tool.md

File: consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/Batch.java (hunk 1 above)
Comment (BUAAserein) [reply consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/Batch.java]: ok

File: consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/LogDispatcher.java (hunk 2 above)
Comment (BUAAserein) [reply consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/LogDispatcher.java]: ok

**Comment by BUAAserein** [comment]
> Excellent Work! Just one problem, Please update the en and zh doc at same time en: docs/UserGuide/Monitor-Alert/Metric-Tool.md zh: docs/zh/UserGuide/Monitor-Alert/Metric-Tool.md

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Pull request merged**